libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx

```diff
@@ -803,11 +818,16 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
         const description = this.getSummaryMessage();
         const attachments = this.getAttachments(this.props.dashboard);
 
+        let unsubscribed: string[] | undefined = undefined;
+        if (editSchedule) {
+            unsubscribed = differenceBy(editSchedule.unsubscribed, toEmails.concat(bccEmails));
```

Comment:
remove entered recipients from "unsubscribed" - allows adding unsubscribed recipients again to the schedule